devtools/conda-recipe/meta.yaml

```diff
@@ -31,14 +31,12 @@ test:
     - nbformat
     - ipykernel
     - ipyext
+    - python-coveralls
+    - pytest-cov
 
-  source_files:
-    - mbuild/examples/*
-
-  commands:
-    - py.test -v --pyargs mbuild
-    - cd mbuild/examples
-    - py.test test_examples.py -v
+#  commands:
+#    - py.test -v --cov=mbuild --cov-report= --pyargs mbuild
+#    - coveralls 
```

Comment:
If we're gonna test this on appveyor, I think it should mirror the travis setup minus the coveralls push but definitely don't leave it turned off. I'm also relatively sure you'll need the `source_files` section.